client/auth/login.jsx

```diff
@@ -135,7 +136,7 @@ export class Auth extends Component {
 								<FormFieldset>
 									<FormTextInput
 										name="auth_code"
-										type="number"
+										type="tel"
```

Comment:
I switched to `tel` because `number` would not allow white spaces.

Reply:
Seems odd to do so when it's not a telephone number, can we use a plain input? @ryelle this would probably read a little funny on VoiceOver wouldn't it?

Reply:
Well, using `tel` will cause the numeric keyboard to appear on mobile devices.

Reply:
It looks like screen readers should still see this as a text input, I don't think it would negatively affect accessibility. except, as mentioned, changing the keyboard -- but an auth code is going to be a number anyway, right?

In testing a `tel`, Safari did try to give me a phone number autofill menu, so maybe turn off autocomplete on this field?

Reply:
Would it help if we did this somewhat similar to how we do credit card formatting? I believe that ends up being a display field and an actual raw value field. This also allows us to show the space as folks type in the auth code.

Reply:
> but an auth code is going to be a number anyway, right?

This code is all digits separated by a white space. Entering a code without any white space is perfectly valid.

> In testing a tel, Safari did try to give me a phone number autofill menu, so maybe turn off autocomplete on this field?

Good catch, this form is missing that attribute ! I'll add it.

> I believe that ends up being a display field and an actual raw value field

Wouldn't that prevent the numeric keyboard to show up?